src/Sylius/Behat/Page/Admin/Order/ShowPage.php

```diff
@@ -265,4 +282,25 @@ private function hasAddress($elementText, $customerName, $street, $postcode, $ci
             (stripos($elementText, $countryName.' '.$postcode) !== false)
         ;
     }
+
+    /**
+     * @param PaymentInterface $payment
+     *
+     * @return NodeElement
+     *
+     * @throws \InvalidArgumentException
+     */
+    private function getLastPaymentElementWithState(PaymentInterface $payment)
+    {
+        /** @var NodeElement[] $paymentElements */
+        $paymentElements = $this->getElement('payments')->findAll('css', '.item');
+
+        foreach ($paymentElements as $paymentElement) {
```

Comment:
it can be omitted if this would work ``->find('css', sprintf('span:contains("%s")', $payment->getState()));``